apps/wing-console/console/server/src/utils/test-runner/simulator-manager.ts

```diff
@@ -0,0 +1,85 @@
+import { mkdtempSync } from "node:fs";
+import { tmpdir } from "node:os";
+import { join } from "node:path";
+
+import uniqueid from "lodash.uniqueid";
+
+import type { Simulator } from "../../wingsdk.js";
+import { createCompiler, type Compiler } from "../compiler.js";
+import { createSimulator } from "../simulator.js";
+
+const getSimulatorInstance = async (simfile: string) => {
+  const stateDir = mkdtempSync(
+    join(tmpdir(), `pwing-console-test-${uniqueid()}`),
+  );
+
+  const testSimulator = createSimulator({
+    stateDir,
+  });
```

Comment:
Maybe use the SDK simulator directly instead of our wrapper. I think we don't need anything from the wrapper.